internal/clients/enx_redirect.go

```diff
@@ -40,6 +42,55 @@ func NewENXRedirectClient(base string, opts ...Option) (*ENXRedirectClient, erro
 	}, nil
 }
 
+func (c *ENXRedirectClient) SendUserReportIndex(ctx context.Context, apikey string, nonce string) error {
```

Comment:
We might want to make a different kind of client for this, `ENXSelfReportClient`, because APIkey should really be a property of the client, not the request.

Reply:
it is a property of the client - changed to use that one.

Reply:
moved to separate client, API is different enough.